service/validate.go

```diff
@@ -93,6 +93,7 @@ func validateBinaryTypes(bins []*pbsubstreams.Binary) error {
 
 		switch wasmCodeTypeID {
 		case "wasm/rust-v1":
+		case "javascript-v8":
```

Comment:
Let's use something more inline with current formatting:

```suggestion
		case "javascript/v8":
```